examples/unet_segmentation_3d.py

```diff
@@ -112,9 +113,15 @@ def _loss_fn(i, j):
 
 @trainer.on(Events.EPOCH_COMPLETED)
 def log_training_loss(engine):
+    writer.add_scalar('Loss/train', engine.state.output, engine.state.epoch)
+    first_image_tensor = engine.state.batch[0][0]
```

Comment:
I think you should show the **model output** instead of the original image, or both.
Just add an output_transform to SupervisedTrainer in this example, the default transform is:
```py
output_transform: Callable = lambda x, y, y_pred, loss: loss.item())
```
You can get all things: _image(x), label(y), output(y_pred)_

Reply:
The model output is now being logged in the latest version.